lib/sequent/util/dry_run.rb

```diff
@@ -186,8 +187,19 @@ def self.these_commands(commands)
                         current_event_store
                       end
 
+        # Set event handlers to classes not instances for dry run. This allows for
+        # variation of class constructor signatures.
+        #
+        # If in test mode use configuration, else set to all classes that inherit from
+        # handler parent classes to encompass possibly nested handlers.
+        event_handler_classes = if ENV['RACK_ENV'] == 'test'
```

Comment:
Why not always use `[*Sequent::Core::Workflows.all, *Sequent::Core::Projectors.all]`?

Reply:
Then the dry run spec would pick up workflows from previous specs

Reply:
We should fix this in the spec (perhaps by stubbing this in the dry_run spec) rather than implementing this in code. Also, not everyone uses `RACK_ENV`, we should get rid of that.